docs/source/ext/connector.rst

```diff
@@ -71,6 +71,11 @@ The ``streamflow.core.deployment`` module defines the ``Connector`` interface, w
     ) -> None:
         ...
 
+    async def get_stream_reader(
```

Comment:
This should have the same indentation as the other methods, and methods should be sorted alphabetically